priv/template/docker-compose.yml.eex

```diff
@@ -10,6 +10,9 @@ services:
       - 8443:8443
     volumes:
       - .:/opt/app
-      - ./deps_docker:/opt/app/deps
-      - ./_build_docker:/opt/app/_build
+      ## uncomment the below lines if you want to see the contents of the
+      ## container's deps and _build directories in your local checkout,
```

Comment:
1) I didn't know how to phrase it. My reasoning was that even if I'm using Docker for local development, I might want to have access to the `_build` and `deps` directories inside the container to debug some issues - see if some modules got compiled or modify some `deps`' source in a quick and dirty way. 

If I felt like `raxx.kit` is hiding those from me and didn't know how to surface them, I might feel like there's too much magic going on. This way they're not confusing to the beginners, but easily discoverable to more advanced people.

2) I've been using "local checkout" with the meaning of "your local copy of some project", from git terminology. It's probably not the best phrase to use here, "host machine" might be better, even if in some cases it might be a very general phrase.

Reply:
I think even though general host is the docker terminology

Reply:
The other reason to mount the directories to your host machine would be to maintain the `_build` directory between `docker-compose down` and `docker-compose up`, otherwise they might have to get built from scratch again.